### PR TITLE
support for python 3.9 (fixes ci issues)

### PR DESCRIPTION
**Pull request by dtebbs**
CI failures related to python 3.9 on mac (spec to force python 3.8 seems to no longer work).
This change adds support for python 3.9 (although this currently requires disabling a pylint check.

See https://github.com/PyCQA/pylint/issues/3882

### CODE REVIEW

File: mpc/.pylintrc [hunk 1]
```diff
@@ -16,6 +16,11 @@ disable=
     protected-access,
     fixme,
     import-outside-toplevel,
+    unsubscriptable-object,
+
+# TODO: `unsubscriptable-object` generates false positives for python 3.9 and pylint==2.6.
+#   https://github.com/PyCQA/pylint/issues/3882
+# Re-enable it when the issue is fixed.
```
Comment (AntoineRondelet) [inline mpc/.pylintrc:23]: The PR: https://github.com/PyCQA/pylint/pull/3890 was merged 2 days ago. Does it fully fix the issue you faced here?

Comment (dtebbs) [reply mpc/.pylintrc:23]: Yes, it appears that should fix it.  I'll watch the repo for the next release and try upgrading when it's available.

Comment (AntoineRondelet) [reply mpc/.pylintrc:23]: Thanks

**Comment by AntoineRondelet** [comment]
LGTM

**Pull request merged**